server/src/modules/stats/boards/boards.stats.service.ts

```diff
@@ -1,4 +1,157 @@
 import { Injectable } from '@nestjs/common';
+import { InjectModel } from '@nestjs/mongoose';
+import { Model, Types } from 'mongoose';
+import { BoardsStats } from 'src/schemas/stats/boards.stats.schema';
 
 @Injectable()
-export class BoardsStatsService {}
+export class BoardsStatsService {
+  constructor(
+    @InjectModel(BoardsStats.name)
+    private readonly boardStatsModel: Model<BoardsStats>,
+  ) {}
+
+  async initStats(boardId: string): Promise<void> {
+    await this.boardStatsModel.create({ boardId: new Types.ObjectId(boardId) });
```

Comment:
i think there is no need to create ObjectId when using `create` or `find` methods - imo its transformed into type defined in the schema. Correct me if i am wrong

Reply:
fixed